### PR TITLE
Display gl points

### PR DESCRIPTION
**Pull request by kerautret**
# PR Description

add a mode to display ball primitive with OpenGL point instead quad mesh.
you can obtain such display (by using key O)
viewer3dballptopengl
or the default one:
viewer3dballdefaultquads

# Checklist

- [na.] Unit-test of your feature with [Catch](http://dgtal.org/doc/stable/moduleCatch.html).
- [x] Doxygen documentation of the code completed (classes, methods, types, members...)
- [x] Documentation module page added or updated.
- [x] New entry in the [ChangeLog.md](https://github.com/DGtal-team/DGtal/blob/master/ChangeLog.md) added.
- [x] No warning raised in Debug ```cmake``` mode (otherwise, Travis C.I. will fail).
- [x] All continuous integration tests pass (Travis & appveyor)

### CODE REVIEW

File: ChangeLog.md [hunk 1]
```diff
@@ -19,6 +19,10 @@
    [#1153](https://github.com/DGtal-team/DGtal/pull/1153))
 
 - *IO Package*
+
+ - Viewer3D: add a mode to display ball primitive with OpenGL point instead quad
+   mesh.
```
Comment (dcoeurjo) [inline ChangeLog.md]: instead of triangulated ball ?

File: src/DGtal/io/doc/moduleDisplay3D.dox [hunk 2]
```diff
@@ -144,6 +144,29 @@ viewer3D.__
 (key C) which can be used in Board2Dto3D described in the following.
 
 
+\subsection  DGtalGLV_Viewer3D_Ball Ball display modes
+
+The Viewer3D class has a special mode to display balls (added from the
+Display3D method \c addBall() ). By default, the balls are constructed
+with OpenGl quad primitive which can be slow if the number of ball is
```
Comment (dcoeurjo) [inline src/DGtal/io/doc/moduleDisplay3D.dox]: quad -> quadrangulated sphere

File: ChangeLog.md (hunk 1 above)
Comment (dcoeurjo) [inline ChangeLog.md]: instead of quadrangulated sphere

**Comment by dcoeurjo** [comment]
Beside my comments. Fine for me

File: ChangeLog.md (hunk 1 above)
Comment (kerautret) [inline ChangeLog.md]: yes quad you are right

File: src/DGtal/io/doc/moduleDisplay3D.dox (hunk 2 above)
Comment (kerautret) [inline src/DGtal/io/doc/moduleDisplay3D.dox]: ok

**Comment by dcoeurjo** [comment]
Perfect. thanks

**Pull request merged**